eventsource/repository_test.go

```diff
@@ -94,6 +103,27 @@ func Test_RepoGetRecords(t *testing.T) {
 	assert.Equal(t, records[1].(*OtherEvent), otherEvent, true)
 }
 
+func Test_RepoGetRecordsByType(t *testing.T) {
+	storeMock, _, serializerMock, aggregatorMock := setupMocks()
+
+	history, baseEvent, _ := createMockDataForLoadAggregate()
+	otherEvent := &OtherEvent{BaseEvent: baseEvent, OtherEventField: 42}
+
+	ctx := context.Background()
```

Comment:
Hmm.. Wondering if we should use context.TODO() for testing.
https://golang.org/src/context/context.go?s=7708:7727#L205